Tests/Runtime/TestLLM.cs

```diff
@@ -449,39 +444,22 @@ public class TestLLMCharacter_Save : TestLLM
         public override LLMCharacter CreateLLMCharacter()
         {
             LLMCharacter llmCharacter = base.CreateLLMCharacter();
-            llmCharacter.save = saveName;
+            llmCharacter.cacheFilename = saveName;
             llmCharacter.saveCache = true;
             return llmCharacter;
         }
 
         public override async Task Tests()
         {
             await base.Tests();
-            TestSave();
+            TestSaveCache();
         }
 
-        public void TestSave()
+        public void TestSaveCache()
         {
-            string jsonPath = llmCharacter.GetJsonSavePath(saveName);
-            string cachePath = llmCharacter.GetCacheSavePath(saveName);
-            Assert.That(File.Exists(jsonPath));
+            string cachePath = llmCharacter.GetCacheSavePath();
```

Comment:
Since this is a unit test for the LLM character we don't really need to worry about the internals of the LLMChatHistory, I've got those covered in the LLMChatHistory tests.